auslib/db.py

```diff
@@ -828,7 +831,7 @@ def _stripNullColumns(self, change):
         # We know a bunch of columns are going to be empty...easier to strip them out
         # than to be super verbose (also should let this test continue to work even
         # if the schema changes).
-        for key in change.keys():
+        for key in change.copy().keys():
```

Comment:
Yes, because the dict size change, when a key is deleted:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: dictionary changed size during iteration
```